### PR TITLE
Updated "What does coala do (for me)?"

### LINKED ISSUE #289: FAQ: Compress "What does coala do (for me)?"
This is too long and should be written shorter and more concise.

difficulty newcomer

Opened via [gitter](https://gitter.im/coala/coala/?at=58245c71df5ae966453c1415) by @sils

### PR DESCRIPTION
**Pull request by Warewolf96**
Fixes #289

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
hey could you please fix the gitmate issues? also, you need a rebase (use ``cobot explain rebase`` on gitter if you need help)

**Comment by Adrianzatreanu** [comment]
your commit message needs  some work too, please see coala.io/commit for more information about our guidelines :) thanks!

**Comment by Warewolf96** [comment]
@Adrianzatreanu sorry about the mistakes. I'm new so still getting a hang of it. I'm trying to rectify them without any delay.

**Comment by Adrianzatreanu** [comment]
no need to hurry ;p this is what newcomer issues are for, you getting accustomed to these problems :) :p
you still have 3 more issues and need a rebase btw

**Comment by srisankethu** [comment]
@Warewolf96 Try running coala from the root directory before submitting  the PR. There is a coafile in the root directory.

**Comment by Warewolf96** [comment]
ohk thanks @srisankethu

**Comment by sils** [comment]
Can you squash your commits together? Use the `git rebase --interactive` - google for it  if you don't know it :)

**Comment by sils** [comment]
and be sure to run coala locally to get rid of all those issues

**Comment by Warewolf96** [comment]
@sils ohk... i'm a little busy for around 6-7 hours, i'll complete this within 10 hours or so.

**Comment by AbdealiJK** [comment]
Read the docs on how to Rebase to avoid the merge commit.
Also gitmate pointed issues

**Comment by Warewolf96** [comment]
Hope this commit is up to the mark...! :)

**Comment by aptrishu** [comment]
commit message needs some changes. Like in shortlog `{filename/module being changed}: {shortlog message}`
Go to https://coala.io/commit .

**Comment by aptrishu** [comment]
Please follow the instructions here https://coala.io/commit carefully. It's like every time you are updating your PR and I'm pointing a new mistake in your commit message. It's pretty simple. Go through that link. and don't worry once you learn these things , you won't be doing it again. bdw we use Imperative mood in commit message. also your shortlog message needs changes.

**Comment by Mixih** [comment]
Make the shortlog clearer too. When you say "compressed" some may think you zipped the files. Instead, you should make it clear the you are making it simpler and more concise.

File: Help/FAQ.rst [hunk 1]
```diff
@@ -8,23 +8,21 @@ What does coala do (for me)?
 ----------------------------
 
 coala is like a spell and grammar checker for source code. Imagine using
-LibreOffice Writer to spellcheck something in English. Why should you have to
-write LibreOfficES to check something written in Spanish?
-That is what coala  tries to fix. Why write the whole thing again if you
-actually just want another analyzing routine? Thats what bears are for. You
-want to use clang or pylint on your project? We got you covered. One command
+LibreOffice Writer to spellcheck something in English but to spellcheck
+something in Spanish is pretty different. That is what coala  tries to fix.
```
Comment (jayvdb) [inline Help/FAQ.rst]: "  " <- two spaces -- needs to be one space.

Comment (jayvdb) [inline Help/FAQ.rst]: " is pretty different."  this doesnt capture the same meaning as "LibreOfficES".

File: Help/FAQ.rst [hunk 2]
```diff
@@ -8,23 +8,21 @@ What does coala do (for me)?
...
+Just use bears for another analyzing routine, without having to write it
+again. You want to use clang or pylint on your project, just one command
 and one configuration to lint all languages in your project.
 
-You have an awesome idea for a new kind of code analysis but don't want to
-write a CLI? Write just the parameters your custom analysis needs and the
-analysis part, we'll take care of the rest. Everything is handled by coala.
+You have an awesome idea for a new kind of code analysis, write just the
```
Comment (jayvdb) [inline Help/FAQ.rst]: "You have an.." -> "If you have an.."

File: Help/FAQ.rst [hunk 3]
```diff
@@ -8,23 +8,21 @@ What does coala do (for me)?
...
+parameters your custom analysis needs and the analysis part, coala will take
+care of the rest.
 
-All in all coala does two things:
+All in all coala makes two things much easier:
```
Comment (jayvdb) [inline Help/FAQ.rst]: "All in all" is superfluous.

But, I think this list is an area where were can reduce the size of the page.  It partially repeats the paragraph above which says "You have an awesome idea for a new kind of code analysis.."

File: Help/FAQ.rst [hunk 4]
```diff
@@ -8,23 +8,15 @@ What does coala do (for me)?
...
-write a CLI? Write just the parameters your custom analysis needs and the
-analysis part, we'll take care of the rest. Everything is handled by coala.
-
-All in all coala does two things:
-
-- Make it easy to use existing static code analyzers by unifying and
-  simplifying the configs
-- Make it easy to write new routines by offering the interface part
-  and everything but the actual analyzer routine.
+If you have an awesome idea for a new kind of code analysis, write just the
```
Comment (yash-nisar) [inline Help/FAQ.rst]: write just the -> just write the

**Comment by Mixih** [comment]
Please fix the gitmate issues

**Comment by Warewolf96** [comment]
@Mixih there is only manual review issue remaining...

File: Help/FAQ.rst [hunk 5]
```diff
@@ -8,23 +8,15 @@ What does coala do (for me)?
...
-analysis part, we'll take care of the rest. Everything is handled by coala.
-
-All in all coala does two things:
-
-- Make it easy to use existing static code analyzers by unifying and
-  simplifying the configs
-- Make it easy to write new routines by offering the interface part
-  and everything but the actual analyzer routine.
+If you have an awesome idea for a new kind of code analysis, just write the
+parameters your custom analysis needs and the analysis part, coala will take
```
Comment (yash-nisar) [inline Help/FAQ.rst]: IMO, this can be shortened to `just write the parameters your custom analysis needs and coala will take care of the analysis part.`

**Comment by sims1253** [comment]
please change the fixes in the commit message to a closes as this is not a bug fix

**Comment by Warewolf96** [comment]
@sils Hey, if u r not busy can u please review this...

**Comment by sils** [comment]
Hi, if you want to speed up reviews, consider helping us reviewing code yourself!

File: Help/FAQ.rst [hunk 6]
```diff
@@ -8,23 +8,15 @@ What does coala do (for me)?
 ----------------------------
 
 coala is like a spell and grammar checker for source code. Imagine using
-LibreOffice Writer to spellcheck something in English. Why should you have to
-write LibreOfficES to check something written in Spanish?
-That is what coala  tries to fix. Why write the whole thing again if you
-actually just want another analyzing routine? Thats what bears are for. You
-want to use clang or pylint on your project? We got you covered. One command
+LibreOffice Writer to spellcheck something in English but to spellcheck
+something in Spanish con't be done in same. That is what coala tries to fix.
```
Comment (sils) [inline Help/FAQ.rst]: typo

**Comment by sils** [comment]
not fully convinced of the summary yet, it's very hard to sum this up properly though :/ maybe the community on the chat can help you create a better one

**Comment by sils** [comment]
I don't have concrete ideas on how to improve other that I think this isn't good enough for most people to understand it.

File: Help/FAQ.rst (hunk 6 above)
Comment (Warewolf96) [inline Help/FAQ.rst]: on it...

**Comment by Warewolf96** [comment]
ok, I'll get help on chat and how to review code proposed by other contributers...?

**Comment by sils** [comment]
sorry, still not a fan of the commit. More text and you remove the part that did the best summary for me previously.

**Pull request closed without merging**